### PR TITLE
Pavlo

### PR DESCRIPTION
**Pull request by PavloTarnovetskyi**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -12,11 +12,11 @@ pipeline {
...
-            steps {
-                cleanWs()
-            }
-        }
+        // stage('Clean Workspace') {
+        //     steps {
+        //         cleanWs()
+        //     }
+        // }
         stage('Cloning our Git '){
```
Comment (nos373737) [inline Jenkinsfile]:
```suggestion
        stage('Cloning our Git'){
```

File: Jenkinsfile [hunk 2]
```diff
@@ -53,9 +53,11 @@ pipeline {
         stage('Ansible connect and deploy Flaskapp'){
             steps{
                 dir('./ansible'){
-                   sh """
-                     ansible-playbook playbook.yml                  
-                    """
+                    retry(10) {
+                         sh """
+                        ansible-playbook playbook.yml
```
Comment (nos373737) [inline Jenkinsfile]:
```suggestion
                             ansible-playbook playbook.yml                  
```

File: Jenkinsfile [hunk 3]
```diff
@@ -53,9 +53,11 @@ pipeline {
         stage('Ansible connect and deploy Flaskapp'){
             steps{
                 dir('./ansible'){
-                   sh """
-                     ansible-playbook playbook.yml                  
-                    """
+                    retry(10) {
+                         sh """
+                        ansible-playbook playbook.yml                  
+                        """
```
Comment (nos373737) [inline Jenkinsfile]:
```suggestion
                               """
```

File: docs/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+What is this change?
```
Comment (nos373737) [inline docs/pull_request_template.md:1]: To every question make them bold. Like "**Question?**"

Comment (nos373737) [reply docs/pull_request_template.md:1]: This can be done by wrapping the text in double star **

File: docs/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+Why is this important?
```
Comment (PavloTarnovetskyi) [inline docs/pull_request_template.md]:
```suggestion
**Why is this important?**
```

File: docs/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+How this has been tested?
```
Comment (PavloTarnovetskyi) [inline docs/pull_request_template.md]:
```suggestion
**How this has been tested?**
```

File: docs/pull_request_template.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+
+
+How can this be rolled back?
+
+Rollback can be performed by reverting these changes in source control and redeploying via standard deployment mechanisms. This change is safe to roll back.
+
+Are there any security risks introduced by this change?
```
Comment (PavloTarnovetskyi) [inline docs/pull_request_template.md]:
```suggestion
**Are there any security risks introduced by this change?**
```

File: Jenkinsfile [hunk 8]
```diff
@@ -53,9 +53,11 @@ pipeline {
         stage('Ansible connect and deploy Flaskapp'){
             steps{
                 dir('./ansible'){
-                   sh """
-                     ansible-playbook playbook.yml                  
-                    """
+                    retry(10) {
+                         sh """
+                             ansible-playbook playbook.yml                  
+                               """
```
Comment (nos373737) [inline Jenkinsfile]:
```suggestion
                           """
```

File: docs/pull_request_template.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+**Why is this important?**
+
+
+
+**How this has been tested?**
+
+
+
+How can this be rolled back?
```
Comment (nos373737) [inline docs/pull_request_template.md]:
```suggestion
**How can this be rolled back?**
```

**Review by nos373737 — APPROVED** [review]
lgtm

**Pull request merged**